### ISSUE TITLE
Revise error of static ports

### ISSUE THREAD

**Issue by hbeni** [HHS81/c182s#505]
See: https://github.com/c172p-team/c172p/issues/1447#issuecomment-1689560624

It looks like we should have the same `error` value for all static ports of the same static system (and consider zero?)

This is pending. Currently we have no idea what that `error` definition really does.
We follow this in the c172p issue item, this here is just a reminder that the c182 is affected too.

**Comment by TheFGFSEagle** [comment]
$FGSource/Systems/static.cxx:100:
```c++
            mach = _mach_node->getDoubleValue();
            pt = p_new * pow(1 + 0.2 * mach*mach*proj_factor*proj_factor, 3.5 );    //total pressure perpendicular to static port (=perpendicular to body x-axis)
            qc_part = (pt - p_new) * _error_factor ;                            //part of impact pressure to be added to static pressure (due to sideslip)
            p_new = p_new + qc_part;
```
Don't really know what it measn though …

**Comment by TheFGFSEagle** [comment]
@davidmegginson

**Comment by TheFGFSEagle** [comment]
So should be actually set to 1 for realistic behaviour ??

**Comment by hbeni** [comment]
> Maybe we should play a little with the tau.

Settled to 0.3 for now after some playing with that.

**Comment by hbeni** [comment]
Closed by 9757568e51e7c482ae7bbf7cfe36dc40c575a041
